Documentation/observability/metrics.rst

```diff
@@ -607,9 +607,9 @@ BGP Control Plane
 ====================== ============================================= ======== ===================================================================
 Name                   Labels                                        Default  Description
 ====================== ============================================= ======== ===================================================================
-``session_state``      ``vrouter``, ``neighbor``                     Enabled  Current state of the BGP session with the peer, Up = 1 or Down = 0
-``advertised_routes``  ``vrouter``, ``neighbor``, ``afi``, ``safi``  Enabled  Number of routes advertised to the peer
-``received_routes``    ``vrouter``, ``neighbor``, ``afi``, ``safi``  Enabled  Number of routes received from the peer
+``session_state``      ``vrouter``, ``neighbor``, ``neighbor_asn``   Enabled  Current state of the BGP session with the peer, Up = 1 or Down = 0
+``advertised_routes``  ``vrouter``, ``neighbor``, ``neighbor_asn``, ``afi``, ``safi``  Enabled  Number of routes advertised to the peer
+``received_routes``    ``vrouter``, ``neighbor``, ``neighbor_asn``, ``afi``, ``safi``  Enabled  Number of routes received from the peer
 ====================== ============================================= ======== ===================================================================
```

Comment:
Please align (vertically) the contents from the different columns, and adjust the delimiters (lines of `=`) accordingly. The preview currently fails to build because the table is not formed correctly.

Reply:
Sorry about that, fixed here: https://github.com/cilium/cilium/pull/35503/commits/08e4332f616899ea869f2ff9e8a24bfba6bf4f71